src/main/java/com/icthh/xm/ms/entity/lep/XmEntityMsLepProcessingApplicationListener.java

```diff
@@ -91,6 +97,7 @@ protected void bindExecutionContext(ScopedContext executionContext) {
         services.put(BINDING_SUB_KEY_PROFILE_EVENT_PRODUCER_SERVICE, profileEventProducer);
         services.put(BINDING_SUB_KEY_COMMENT_SERVICE, commentService);
         services.put(BINDING_SUB_KEY_PERMISSION_SERVICE, permissionCheckService);
+        services.put(BIND_KEY_FUNCTION_SERVICE, functionService);
```

Comment:
remove functionService from lepContext
it's wron way
use lep commos for instead